parking-lot-help-request/README.md

```diff
@@ -0,0 +1,119 @@
+
+* [Purchase the necessary hardware and configure it](#hardware).
+* [Configure Notehub](#notehub).
+* [Configure the project's Notecard](#notecard-firmware).
+* [Set up a Twilio account and configure the data to route from Notehub to Twilio](#twilio-firmware).
+## Hardware
+
+The following hardware is required to run the Parking Lot Help Request application.
+
+* [Notecarrier A with pre-soldered headers](https://shop.blues.io/products/carr-al)
+* [Cellular Notecard](https://shop.blues.io/collections/notecard) (LTE Cat-1 versions are highly recommended)
+* [Sparkfun RGB LED breakout - WS2812B](https://www.sparkfun.com/products/13282)
+* [Twidec 12mm momentary push button with pre-soldered wires](https://www.amazon.com/gp/product/B08JHW8BPV/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&th=1)
+* [LiPo battery](https://www.adafruit.com/product/2011)
+* [Solar panel with JST connectors (any of these are compatible)](https://www.seeedstudio.com/catalogsearch/result/?q=solar%20panels) (This [3W model](https://www.seeedstudio.com/3W-Solar-Panel-138X160.html) is the one used in the project)
+* [Breadboard](https://www.adafruit.com/product/64)
+* [Male-to-male jumper wires](https://www.adafruit.com/product/758)
+* [Break-away 0.1" pin strip male headers](https://www.adafruit.com/product/392)
+
+### Hardware Assembly 
+
+1. After purchasing your hardware, use the [Notecard and Notecarrier Quickstart documentation](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/) to assemble your Notecarrier A and Notecard - don't worry too much over connecting to Notehub: this document will cover it shortly.
+1. Break off 3 male pins in a strip and solder to each side of the Sparkfun LED breakout. (This [video from Adafruit](https://www.youtube.com/watch?v=Z0joOKaQ43A&ab_channel=AdafruitIndustries) provides a good example of how to solder male pins to a board like the one the LED comes in.)
+![NeoPixel with male pins soldered onto the underside of it](images/readme-soldered-led.png)
+_These pins will allow the LED to sit in a breadboard and easily interact with the Notecarrier through jumper wires._
+1. Cut a male jumper wire in half and use a wire stripper to strip at least 1/2" of coating off of both the ends of the jumper wire and each wire connected to the Twidec push button. Also snip off the soldered tip of the button's wires so the filaments are free floating.
+![Twidec button and jumper wire both stripped down to expose their wires for soldering](images/readme-wires-to-solder.jpg)
+_Notice that both the button and jumper wires are exposed for soldering, also there's a length of heat shrink on both button wires that will be used to cover the soldered portion of wire later._
+
+1. Use the "Western Union" X-style of wire wrapping to join the exposed jumper wires together with the Twidec wires and solder them together.
```

Comment:
```suggestion
1. Use the "Western Union" X-style of wire splicing to wrap the exposed jumper wires together with the Twidec wires and solder them together.
```

maybe include a photo like this so one doesn't have to google it.
![image](https://user-images.githubusercontent.com/22032748/198330691-a0041c2c-06ff-49fb-b6d0-c58614881e6e.png)

Reply:
Good suggestion. Included, thanks